WalletWasabi/Wallets/Wallet.cs

```diff
@@ -550,7 +553,7 @@ public void SetWaitingForInitState()
 	public static Wallet CreateAndRegisterServices(Network network, BitcoinStore bitcoinStore, KeyManager keyManager, WasabiSynchronizer synchronizer, string dataDir, ServiceConfiguration serviceConfiguration, HybridFeeProvider feeProvider, IBlockProvider blockProvider)
 	{
 		var wallet = new Wallet(dataDir, network, keyManager);
-		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider);
+		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider, new());
```

Comment:
Nit:
```suggestion
		wallet.RegisterServices(bitcoinStore, synchronizer, serviceConfiguration, feeProvider, blockProvider, new CoinPrison());
```

I know we love to write as few characters as possible, but a simple new() doesn't tell me anything in this context.

Reply:
yes please. Imagine this `var mb = new MessageBroker(new(), new(), new(), false, null, new())` wtf is that!?